src/Illuminate/Database/Eloquent/Concerns/HasUlids.php

```diff
@@ -0,0 +1,64 @@
+<?php
```

Comment:
Is this now built in functionality as of 9.30? 

https://github.com/laravel/framework/pull/44074